pkg/store/prometheus_test.go

```diff
@@ -398,83 +398,83 @@ rule_files:
+					File:                              fmt.Sprintf("%s/../../examples/alerts/alerts.yaml", curr),
 					Rules:                             []*storepb.Rule{someAlert, someAlert},
 					Interval:                          60,
 					DeprecatedPartialResponseStrategy: storepb.PartialResponseStrategy_WARN, PartialResponseStrategy: storepb.PartialResponseStrategy_WARN,
 				},
 				{
 					Name:                              "thanos-store.rules",
-					File:                              "/home/bwplotka/Repos/thanos/pkg/store/../../examples/alerts/alerts.yaml",
+					File:                              fmt.Sprintf("%s/../../examples/alerts/alerts.yaml", curr),
 					Rules:                             []*storepb.Rule{someAlert, someAlert, someAlert, someAlert},
 					Interval:                          60,
 					DeprecatedPartialResponseStrategy: storepb.PartialResponseStrategy_WARN, PartialResponseStrategy: storepb.PartialResponseStrategy_WARN,
 				},
 				{
 					Name:                              "thanos-bucket-replicate.rules",
-					File:                              "/home/bwplotka/Repos/thanos/pkg/store/../../examples/alerts/rules.yaml",
+					File:                              fmt.Sprintf("%s/../../examples/alerts/rules.yaml", curr),
 					Rules:                             []*storepb.Rule{},
 					Interval:                          60,
 					DeprecatedPartialResponseStrategy: storepb.PartialResponseStrategy_WARN, PartialResponseStrategy: storepb.PartialResponseStrategy_WARN,
 				},
 				{
 					Name:                              "thanos-query.rules",
-					File:                              "/home/bwplotka/Repos/thanos/pkg/store/../../examples/alerts/rules.yaml",
+					File:                              fmt.Sprintf("%s/../../examples/alerts/rules.yaml", curr),
 					Rules:                             []*storepb.Rule{someRecording, someRecording, someRecording, someRecording, someRecording},
 					Interval:                          60,
 					DeprecatedPartialResponseStrategy: storepb.PartialResponseStrategy_WARN, PartialResponseStrategy: storepb.PartialResponseStrategy_WARN,
 				},
 				{
-					Name: "thanos-receive.rules", File: "/home/bwplotka/Repos/thanos/pkg/store/../../examples/alerts/rules.yaml",
+					Name: "thanos-receive.rules", File: fmt.Sprintf("%s/../../examples/alerts/rules.yaml", curr),
 					Rules:                             []*storepb.Rule{someRecording, someRecording, someRecording, someRecording, someRecording, someRecording},
 					Interval:                          60,
 					DeprecatedPartialResponseStrategy: storepb.PartialResponseStrategy_WARN, PartialResponseStrategy: storepb.PartialResponseStrategy_WARN,
 				},
 				{
 					Name:                              "thanos-store.rules",
-					File:                              "/home/bwplotka/Repos/thanos/pkg/store/../../examples/alerts/rules.yaml",
+					File:                              fmt.Sprintf("%s/../../examples/alerts/rules.yaml", curr),
```

Comment:
We don't need curr technically.. We can do just `./../../examples/alerts/rules.yaml` I believe.